Testreglar/1.3.1/Nett/nett-1.3.1e.json

```diff
@@ -0,0 +1,139 @@
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testside har ikke skjema."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hva skjema tester du?",
+			"ht": "<p>Beskriv skjemaet, slik at det er mulig å identifisere det i ettertid. Om du finner flere skjema, registrerer du et og et.</p>",
+			"type": "tekst",
+			"label": "Skjema:",
+			"multilinje": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Finnes det skjemaelementer med visuelle instruksjoner?",
+			"ht": "<p>Gjør en visuell inspeksjon og sjekk om det finnes skjemaelementer med visuelle instruksjoner/hjelpetekst på testsiden.</p>\n<p>Instruksjoner/hjelpetekst gir vanligvis mer informasjon om hva eller hvordan brukeren skal fylle ut enkelte skjemaelementer eller grupper av skjemaelementer. Instruksjoner kan inneholde informasjon om format, tips eller mer utfyllende forklaring av hva som skal fylles ut. Plasseringen kan for eksempel være</p>\n<ul>\n<li>en del av ledeteksten</li>\n<li>rett over eller under skjemaelementet</li>\n<li>ved siden av skjemaelementet</li>\n<li>i pekerfølsomt innhold </li>\n</ul>\n<p><strong>Merk: </strong>Du skal ikke teste følgende:</p>\n<ul>\n<li>overordnet informasjon, som er felles for hele skjemaet, og vanligvis ligger utenfor selve skjema dvs. <code>&lt;form&gt;</code>-element.</li>\n<li>placeholdertekst</li>\n<li>title-tekst</li>\n<li>løpende tekst mellom skjemaelementer</li>\n</ul>\n<p><strong>Merk:</strong> SK 1.3.1 krever ikke at det finnes ledetekst eller instruksjoner. Dette testes i SK 3.3.2.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Skjema har ikke visuelle instruksjoner."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken instruksjon tester du?",
+			"ht": "<p>Beskriv instruksjonen, slik at det er mulig å identifisere den i ettertid. Om du finner flere instruksjoner, registrerer du en og en.</p>",
+			"type": "tekst",
+			"label": "Instruksjon:",
+			"multilinje": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Har instruksjonen en tydelig visuell relasjon til et skjemaelement eller en gruppe skjemaelementer?",
+			"ht": "<p>Hvis instruksjonen har en tydelig visuell relasjon til et skjemaelement eller en gruppe av skjemaelementer, skal dette også ivaretas i koden.</p>\n<p>Slik tester du:</p>\n<ul>\n<li>Gjør en skjønnsmessig vurdering av om det er tydelig visuell relasjon mellom instruksjon og aktuell skjemaelement.</li>\n<li>I vurderingen kan du for eksempel vektlegge\n<ul>\n<li>instruksjonens plassering</li>\n<li>bruk av visuelle virkemidler som farger, rammer, ikoner</li>\n<li>tekstlig informasjon i instruksjon som gjengir et bestemt skjemaelement, for eksempel passordfelt</li>\n</ul>\n</li>\n</ul>",
```

Comment:
Her bør vi spesifisere enda mer av hva som er tydelig visuell relasjon. Siste del er veldig bra mens første del gir veldig mye rom for skjønnsmessig vurdering som kan gjøre at vi tester forskjellig.